dotcom-rendering/src/web/server/frontToHtml.tsx

```diff
@@ -1,79 +1,145 @@
+		// 'https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-RegularItalic.woff2',
+		'https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-Bold.woff2',
+		'https://assets.guim.co.uk/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-Regular.woff2',
+		// 'http://assets.guim.co.uk/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-RegularItalic.woff2',
+		'https://assets.guim.co.uk/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-Bold.woff2',
+	];
+
+	const polyfillIO =
+		'https://assets.guim.co.uk/polyfill.io/v3/polyfill.min.js?rum=0&features=es6,es7,es2017,es2018,es2019,default-3.6,HTMLPictureElement,IntersectionObserver,IntersectionObserverEntry,URLSearchParams,fetch,NodeList.prototype.forEach,navigator.sendBeacon,performance.now,Promise.allSettled&flags=gated&callback=guardianPolyfilled&unknown=polyfill&cacheClear=1';
+
+	/**
+	 * The highest priority scripts.
+	 * These scripts have a considerable impact on site performance.
+	 * Only scripts critical to application execution may go in here.
+	 * Please talk to the dotcom platform team before adding more.
+	 * Scripts will be executed in the order they appear in this array
+	 */
+	const priorityScriptTags = generateScriptTags([
+		{ src: polyfillIO },
+		...getScriptArrayFromFile('bootCmp.js'),
+		...getScriptArrayFromFile('ophan.js'),
+		Front.config && { src: Front.config.commercialBundleUrl },
+		...getScriptArrayFromFile('sentryLoader.js'),
+		...getScriptArrayFromFile('dynamicImport.js'),
+		...getScriptArrayFromFile('islands.js'),
+	]);
+
+	/**
+	 * Low priority scripts. These scripts will be requested
+	 * asynchronously after the main HTML has been parsed. Execution
+	 * order is not guaranteed. It is even possible that these execute
+	 * *before* the high priority scripts, although this is very
+	 * unlikely.
+	 */
+	const lowPriorityScriptTags = generateScriptTags([
+		...getScriptArrayFromFile('atomIframe.js'),
+		...getScriptArrayFromFile('embedIframe.js'),
+		...getScriptArrayFromFile('newsletterEmbedIframe.js'),
+		...getScriptArrayFromFile('relativeTime.js'),
+	]);
+
+	const gaChunk = getScriptArrayFromFile('ga.js');
+	const modernScript = gaChunk.find((script) => script?.legacy === false);
+	const legacyScript = gaChunk.find((script) => script?.legacy === true);
+	const gaPath = {
+		modern: modernScript?.src as string,
+		legacy: legacyScript?.src as string,
+	};
+
+	/**
+	 * We escape windowGuardian here to prevent errors when the data
+	 * is placed in a script tag on the page
+	 */
+	const windowGuardian = escapeData(
+		JSON.stringify(makeFrontWindowGuardian(Front)),
+	);
+
+	const keywords =
+		typeof Front.config.keywords === 'undefined'
+			? ''
+			: Front.config.keywords;
```

Comment:
This can be replaced inline with a nullish coalescing operator:

```suggestion
	const keywords = Front.config.keywords ?? '';
```